invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/customFields/Thesis.js

```diff
@@ -31,7 +31,7 @@ export class Thesis extends Component {
         <Grid padded>
           <Grid.Column width="16">
             <Input
-              fieldPath={fieldPath}
+              fieldPath={`${fieldPath}.university`}
```

Comment:
ping @alejandromumo because I don't remember the outcome for thesis fields
shouldn't this path be set on the parent of this component ?

Reply:
![Screenshot 2023-09-19 at 16 29 42](https://github.com/inveniosoftware/invenio-rdm-records/assets/51617644/81e586c3-0457-4dfd-add6-9afeca76cc49)

Reply:
yeah I think this path is set on the parent. 

What was the issue there? I only see one occurrence of `thesis:thesis` in the codebase and it's on ZenodoRDM (it's likely wrong, had to investigate)